.github/workflows/build.yml

```
@@ -69,6 +69,17 @@ jobs:
         with:
           dotnet-version: 7.0.x
 
+      - name: Install .NET 8.0
+        uses: actions/setup-dotnet@v1
```

actionlint (1.7.7)

the runner of "actions/setup-dotnet@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)